src/main/kotlin/step2/Operator.kt

```diff
@@ -0,0 +1,18 @@
+package step2
+
+enum class Operator(val operator: String, val operation: (Int, Int) -> Int) {
+    PLUS("+", operation = { x, y -> x + y }),
+    MINUS("-", operation = { x, y -> x - y }),
+    MULTIPLY("*", operation = { x, y -> x * y }),
+    DIVIDE("/", operation = { x, y -> x / y });
+
+    companion object {
+        fun from(operator: String): Operator? {
+            return values().find { it.operator == operator }
+        }
```

Comment:
굳이 사용처에서 예외처리 할 필요없이 메서드 파라미터로 받은 operator를 Operator 객체로 변환할 때, 
일치하는 케이스가 없으면 바로 예외가 발생하도록 처리할 수 있을 것 같아요. 😃

```suggestion
        fun from(operator: String): Operator {
            return values().find { it.operator == operator } ?: throw IllegalArgumentException("예외 메시지")
        }
```

Reply:
이렇게 되면 Non Null인 값만 반환하도록 보장할 수 있군요! 이대로 수정하겠습니다!